projects/plugins/starter-plugin/composer.json

```diff
@@ -12,7 +12,8 @@
 		"automattic/jetpack-identity-crisis": "@dev",
 		"automattic/jetpack-my-jetpack": "@dev",
 		"automattic/jetpack-plugins-installer": "@dev",
-		"automattic/jetpack-sync": "@dev"
+		"automattic/jetpack-sync": "@dev",
+		"automattic/jetpack-connection": "@dev"
```

Comment:
Could you move it higher so the list remains alphabetically ordered?

Reply:
Alright I just added it with `composer require`, not manually